avm/res/resources/resource-group/main.bicep

```
@@ -0,0 +1,131 @@
+metadata name = 'Resource Groups'
+metadata description = 'This module deploys a Resource Group.'
+metadata owner = 'Azure/module-maintainers'
+
+targetScope = 'subscription'
+
+@description('Required. The name of the Resource Group.')
+param name string
+
+@description('Optional. Location of the Resource Group. It uses the deployment\'s location when not provided.')
+param location string = deployment().location
+
+@description('Optional. The lock settings of the service.')
+param lock lockType
+
+@description('Optional. Array of role assignment objects that contain the \'roleDefinitionIdOrName\' and \'principalId\' to define RBAC role assignments on this resource. In the roleDefinitionIdOrName attribute, you can provide either the display name of the role definition, or its fully qualified ID in the following format: \'/providers/Microsoft.Authorization/roleDefinitions/c2f4ef07-c644-48eb-af81-4b1b4947fb11\'.')
+param roleAssignments roleAssignmentType
```

```suggestion
@description('Optional. Array of role assignments to create.')
param roleAssignments roleAssignmentType
```
Update to the new version of the interface for Role Assignment  [Link](https://azure.github.io/Azure-Verified-Modules/specs/shared/interfaces/#role-assignments). Note this will need a readme regeneration.